docs/extending/extending_client_side.md

```diff
@@ -25,21 +25,21 @@ The simplest way is to add global JavaScript files via hooks, see [](insert_edit
 
 For JavaScript added when a specific Widget is used you can add an inner `Media` class to ensure that the file is loaded when the widget is used, see [Django's docs on their form `Media` class](https://docs.djangoproject.com/en/stable/topics/forms/media/#assets-as-a-static-definition).
 
-In a similar way, Wagtail's [](./template_components) provide a `media` property or `Media` class to add scripts when rendered.
+In a similar way, Wagtail's [](./template_components) provides a `media` property or `Media` class to add scripts when rendered.
```

Comment:
I think the singular form is correct here. Assuming the title for the template component is 'template components'. 

This will currently render as 'Wagtail's template components provide a media property...'. 

I think this is better than  'Wagtail's template components provides a media property...'.